lucee-java/lucee-core/src/lucee/runtime/text/xml/XMLUtil.java

```diff
@@ -250,11 +250,13 @@ public static final Document parse(InputSource xml,InputSource validator, boolea
             else {
             	XMLUtil.setAttributeEL(factory,XMLConstants.VALIDATION_SCHEMA, Boolean.TRUE);
             	XMLUtil.setAttributeEL(factory,XMLConstants.VALIDATION_SCHEMA_FULL_CHECKING, Boolean.TRUE);
-            
-                
             }
-            
-            
+
+            XMLUtil.setAttributeEL(factory,XMLConstants.EXTERNAL_GENERAL_ENTITIES, Boolean.FALSE);
```

Comment:
Sounds like normally you would also set `disallow-doctype-decl` to `false` but that broke stuff when installing a site during service to service communication.  This other items do prevent the problem.  See https://github.com/MasterControlInc/mastercontrol/pull/2041/files for the unit test with both bad cases.